src/kohlrahbi/__init__.py

```diff
@@ -104,56 +106,93 @@ def load_all_known_pruefis_from_file(
 
-    if not assume_yes:
-        check_output_path(path=output_path)
-    else:
-        if output_path.exists():
-            click.secho(f"The output directory '{output_path}' exists already.", fg="yellow")
-        else:
-            output_path.mkdir(parents=True)
-            click.secho(f"I created a new directory at {output_path}", fg="yellow")
+    change_history_collection: dict[str, pd.DataFrame] = {}
+
+    for ahb_file_path in ahb_file_paths:
+        # extract the sheet name from the file name
+        # COMDISAHB-informatorischeLesefassung1.0e_99991231_20240401.docx -> COMDISAHB
+        sheet_name = create_sheet_name(ahb_file_path.name)
+
+        doc = docx.Document(ahb_file_path)  # Creating word reader object.
+        logger.info("🤓 Start reading docx file '%s'", str(ahb_file_path))
+        change_history_table: ChangeHistoryTable = get_change_history_table(document=doc)
+
+        if change_history_table is None:
+            continue
+        change_history_table.sanitize_table()
+
+        change_history_collection[sheet_name] = change_history_table.table
+
+    # add timestamp to file name
+    # there are two timestamps: one with datetime and another one with just date information.
+    # It is handy during debugging to save different versions of the output files with the datetime information.
+    # But in production we only want to save one file per day.
+    # current_timestamp = datetime.now().strftime("%Y-%m-%dT%H-%M-%S")
+    current_timestamp = datetime.now().strftime("%Y-%m-%d")
+    path_to_change_history_excel_file = output_path / f"{current_timestamp}_change_histories.xlsx"
+
+    logger.info("💾 Saving change histories xlsx file %s", path_to_change_history_excel_file)
+
+    # Define column widths (example: 20, 15, 30, etc.)
+    column_widths = [6, 6, 46, 52, 52, 38, 15]  # Replace with your desired widths
+
+    # Create a Pandas Excel writer using XlsxWriter as the engine
+    # https://github.com/PyCQA/pylint/issues/3060 pylint: disable=abstract-class-instantiated
+    with pd.ExcelWriter(path_to_change_history_excel_file, engine="xlsxwriter") as writer:
+        for sheet_name, df in change_history_collection.items():
+            df.to_excel(writer, sheet_name=sheet_name)
+
+            # Access the XlsxWriter workbook and worksheet objects
+            workbook = writer.book
+            worksheet = writer.sheets[sheet_name]
+
+            # Create a text wrap format
+            wrap_format = workbook.add_format({"text_wrap": True})
+
+            # Get the dimensions of the DataFrame
+            (max_row, max_col) = df.shape
+
+            assert max_col + 1 == len(column_widths)  # +1 cause of index
+
+            # Apply text wrap format to each cell
+            for col_num, width in enumerate(column_widths):
+                worksheet.set_column(col_num, col_num, width, wrap_format)
+
+
+def scrape_pruefis(pruefis: list[str], input_path: Path, output_path: Path, file_type: list[str]):
```

Comment:
du könntest den file-type als `Literal["flatahb", "csv", "xlsx", "conditions"]` typisieren, dann ist weniger rätselraten als bei einem string

Reply:
done in 041a02d53c242aa332ce1080733f1954f5592c1d